File: Capitulo1/1.3Exponenciales/exponencial.py
t=np.arange(0,5,0.001); #Dominio
#misma base, diferente exponente
a=2;
b1=1/3;
b2=1/2;
b3=2;
b4=3;
y1=a**(b1*t); 
y2=a**(b2*t);
y3=a**(b3*t);
y4=a**(b4*t); 
plt.subplot(2,2,1),plt.plot(t,y1), plt.title('Exponente=' +str(b1))
plt.subplot(2,2,2),plt.plot(t,y2), plt.title('Exponente=' +str(b2))
plt.subplot(2,2,3),plt.plot(t,y3), plt.title('Exponente=' +str(b3))
plt.subplot(2,2,4),plt.plot(t,y4), plt.title('Exponente=' +str(b4))

File: Capitulo1/1.3Exponenciales/exponencial.m
t=0:0.001:5; %Dominio
%misma base, diferente exponente
a=2;
b1=1/3;
b2=1/2;
b3=2;
b4=3;
y1=a.^(t*b1); 
y2=a.^(t*b2);
y3=a.^(t*b3);
y4=a.^(t*b4); 
subplot(2,2,1),plot(t,y1), title('Exponente=', num2str(b1))
subplot(2,2,2),plot(t,y2), title('Exponente=', num2str(b2))
subplot(2,2,3),plot(t,y3), title('Exponente=', num2str(b3))
subplot(2,2,4),plot(t,y4), title('Exponente=', num2str(b4))
